README.md

```diff
@@ -2,14 +2,14 @@
 
 ![mainbanner](https://user-images.githubusercontent.com/32886642/209586234-44f0d161-fe39-40c7-9ea8-6fc106547fd7.png)
 
-TGC is a complete game overhaul that started as attempt to merge all the various disparate HFM and HPM based submods into a single modding experience, that objective has been reached long ago. Nowadays, the team aims to improve historical accuracy and detail, adding plausible alt history paths and flavor to all nations, and designing new mechanics to make Victoria 2 feel new again. 
+TGC is a complete game overhaul that started as an attempt to merge all the various disparate HFM and HPM based submods into a single modding experience, this objective has been reached long ago. Nowadays, the team aims to improve historical accuracy and detail, adding plausible alt-history paths and flavor to all nations, and designing new mechanics to make Victoria 2 feel new again.
```

Comment:
Suggestion:
improve historical accuracy and detail by adding plausible alt-history paths, immersive flavor to all nations, and designing new mechanics to make Victoria 2 feel new and refreshed again.